app/views/users/_review.html.haml

```diff
@@ -0,0 +1,8 @@
+%article.review
+  %li.row
+    .col-sm-2
+      %p
+        = link_to "\"#{review.video_title}\"", video_path(review.video)
```

Comment:
This should be without string interpolation, `review.video_title` will return a string right. Take a look at above comments.